### DISCUSSION TITLE
Oobabooga's text-generation-webui has changed characters file format from JSON to YAML - New maintainer for this extension wanted.

### DISCUSSION THREAD

**Discussion by theubie** [theubie/complex_memory#4] [Announcements]
So, I know I'm a very small player in this software field so this isn't a big deal in the grand scheme of things, but I will not be maintaining this extension any longer.

I hate Python as a general rule, but I use it when I have to.  

I loathe YAML with the burning passion of a million sons.  I won't go into detail, but YAML cost me about $10,000 to a client years ago, and I have vowed to never use or be part of any software that uses it again.  

So, with that said, if people want to continue to use this extension, awesome!  It will need someone new to take over, though.

Thanks to those who contributed and offered great suggestions.

**Comment by remghoost** [comment]
I'm pretty new to programming (learning for about 6 months now), but I'd be willing to adapt this over to YAML in the next day or two. 

I'd more than likely have to re-read through all of the code to make sure I understand it all as well. I might have a question or two to ask on it, if you'd be okay with that.

Not exactly sure what I could contribute past that, but I'd be willing to approve pull requests and generally keep the repo alive.

My condolences on your experiences with YAML and thank you for your contributions to the LLM space!
